.pre-commit-config.yaml

```diff
@@ -35,7 +35,7 @@ repos:
       - id: pyupgrade
         args: ['--py310-plus']
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
[F] I couldn't get the pre-commit to run without making this upgrade. I am happy to pull it out into a separate PR if needed 🙏🏻.

Reply:
That's fine. Thank you for updating it.